how-to/join_kubeflow_ecosystem.md

```diff
@@ -0,0 +1,144 @@
+# Application for a Project to Join the Kubeflow Ecosystem
+
+Please see the [proposals/new-project-join-process.md](Documentation) to
+better understand the full process for submitting a new project.
+In short, copy this Application Template and populate the document.
+--!>
+
+## CNCF checklist
+- [ ] All project metadata and resources are vendor-neutral
+- [ ] Governance structure
+- [ ] Contributing guides
+- [ ] Public list of adopters.
+
+
+## Background information
+
+1. Submitter Name
+    -
+
+1. Submitter’s relationship to project / title
+    -
+
+1. Project Name
+    -
+
+1. Why is this project is valuable to the Kubeflow Community?
+    -
+
+1. Why is it beneficial for this project to be a part of the Kubeflow Community?
+    - 
+
+1. List of integrations with Kubeflow Core components (and potential)
```

Comment:
```suggestion
1. List of existing (and potential) integrations with Kubeflow Core components
```

Reply:
I can't recall, do we explicitly define "core" components anywhere?

Reply:
I think we're actually looking to get rid of that language.